File: Score High.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int n,i,max=1;
        scanf("%d",&n);
        int a[4];
        for(i=0;i<4;i++)
        {
            scanf("%d",&a[i]);
            if(max<a[i])
            max=a[i];
        }
        printf("%d\n",max);
    }
}

File: Score High.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int n=sc.nextInt();
            int [] a = new int[ 4 ];
            int max=0;
            for(int i=0;i<4;i++)
            {
                a[i]=sc.nextInt();
                if(max<a[i])
                max=a[i];
            }
            System.out.println(max);
        }
    }
}